### ISSUE TITLE
How to load text file via relative path?

### ISSUE THREAD

**Issue by dnltsk** [kscripting/kscript#399]
I want to open files that are located beside the main script file.
```kotlin
@file:Import("Models.kts")
val data = Gson().fromJson(FileReader("data.json"), Models.Data::class.java)
```
fails with 
```
java.io.FileNotFoundException: config.live.json (No such file or directory)
```
Using he absolute path works straight away. Cannot find anything guidance in the documentation.

**Comment by dnltsk** [comment]
The sample code above is working finr when I execute the script from console like
```
> ./Main.kts
```

But not when executed from within IntelliJ, when the project was created via
```
> kscript --idea Main.kts
```
Then, it expects to find the text file inside the `.../Library/Caches/kscript/idea_59...` path.

**Comment by aartiPl** [comment]
Yes, because --idea creates IntelliJ project, which is intended for debugging, not as a runtime environment. The project is located in the cache, so you must copy the data file if you want to use it.
